src/QMCWaveFunctions/tests/test_einset.cpp

```diff
@@ -205,6 +205,159 @@ TEST_CASE("Einspline SPO from HDF diamond_1x1x1", "[wavefunction]")
 
 #endif
 
+#if !defined(QMC_COMPLEX)
+  /* 
+     Here we test the low-level spline implementation of applyRotation().
+
+     Manually encode the unitary transformation. Ugly, but it works.
+     @TODO: Use the total rotation machinery when it's implemented
+
+     NB: This is truncated to 5 sig-figs, so there is some slop here as
+         compared to what is done in the splines via apply_rotation().
+	 So below we reduce the threshold for comparison. This can
+	 probably be ditched once we have a way to grab the actual 
+	 rotation matrix...
+  */
+  const auto orbitalsetsize = spo->getOrbitalSetSize();
```

Comment:
Add a check `REQUIRE(orbitalsetsize == 8));` will be helpful

Reply:
Done.